### DISCUSSION TITLE
Monorepo support

### DISCUSSION THREAD

**Discussion by emily-malone** [vuejs/vitepress#579] [Q&A]
Can vitepress support rendering a tree structure from multiple docs folders in a monorepo?

**Comment by lovetingyuan** [comment]
You can try this,

```html
### my-package

<CL />

<script setup>
  import CL from '@root/packages/your-package/README.md'
</script>

```
`@root` is an alias configured in vite field of .vitepress/config.ts, means the root location of your monorepo.

**Comment by lovetingyuan** [comment]
I found a simper way to do this, 

Just use `@include`, for example: `<!-- @include: ../packages/foo.md -->`

But I did not found this usage in vitepress document site, so maybe this way will be changed in the future.

**Reply by borisdamevin** [reply]
@yyx990803 This are the right way for Dynamic Markdown? _(like MDX)_

Thank you.
